src/.clwb/.bazelproject

```diff
@@ -0,0 +1,20 @@
+directories:
+  # Add the directories you want added as source here
+  # By default, we've added your entire workspace ('.')
+  .
+
+# Automatically includes all relevant targets under the 'directories' above
+derive_targets_from_directories: true
+
+targets:
+  # We add all the targets here to automatically load all possible symbols.
+  # This can slow down symbol loading, but it's much more reliable then picking
+  # and choosing targets.
+  //...
+
+additional_languages:
+  # Uncomment any additional languages you want supported
+  # dart
+  # javascript
+  # python
```

Comment:
Might as well uncomment python since we have some of that

Reply:
Done, @akhilveeraghanta - just FYI for when we do python-bazel integration (not sure what that looks like, but figured it's worth noting).